ruff.toml

```diff
@@ -26,6 +26,7 @@ fix = true
 
 [per-file-ignores]
 "src/universal_transfer_operator/__init__.py" = ["F401"]
+"src/universal_transfer_operator/integrations/hooks/fivetran.py" = ["C901"]
```

Comment:
Why are we doing this? - https://www.flake8rules.com/rules/C901.html
Which function is too complex?

Reply:
This is hook code and this function in hook is too complex.